wpinet/src/main/java/edu/wpi/first/net/WPINetJNI.java

```diff
@@ -80,6 +80,22 @@ public static synchronized void forceLoad() throws IOException {
    */
   public static native void removePortForwarder(int port);
 
+  /**
+   * Create a web server at the given port. Note that local ports less than 1024 won't work as a
```

Comment:
Should probably add a note about which ports are open in an FMS setting to minimize cases of teams running into blocked ports at competition.